### ISSUE TITLE
Feature request: Support "import PropTypes from 'prop-types'"

### ISSUE THREAD

**Issue by ekoneko** [KnisterPeter/react-to-typescript-definitions#439]
For example

````js
import PropTypes from 'prop-types';
Component.propTypes = {
  optionalBool: PropTypes.bool
}
````
We got `optionalBool?: any;`, not bool.

**Comment by FDiskas** [comment]
Yes, that would be nice :page_facing_up:

**Comment by KnisterPeter** [comment]
Released in 0.23

**Comment by KnisterPeter** [comment]
Also note that it must be
```ts
import  * as PropTypes from 'prop-types';
```
for typescript
